docs/conf.py

```diff
@@ -186,10 +185,7 @@
 #
 # TODO: Remove or adjust the ACME entry after you update the contributing guide
 
-linkcheck_ignore = [
-    "http://127.0.0.1:8000",
-    "https://github.com/canonical/ACME/*"
-]
+linkcheck_ignore = ["http://127.0.0.1:8000", "https://github.com/canonical/ACME/*"]
```

Comment:
I would revert this change, purely because `linkcheck_ignore` is used extensively across many projects and requires many inputs.

```suggestion
linkcheck_ignore = [
    "http://127.0.0.1:8000",
    "https://github.com/canonical/ACME/*"
    ]
```